yt/frontends/ramses/particle_handlers.py

```diff
@@ -18,20 +19,24 @@ def register_particle_handler(ph):
     PARTICLE_HANDLERS.add(ph)
 
 
-class RAMSESParticleFileHandlerRegistry(type):
+class RAMSESParticleFileHandlerRegistry(abc.ABCMeta):
```

Comment:
This patterns looks a lot like the `RegisteredDataset` metaclass and the `Dataset` class, but I wasn't sure what this did before inspecting the code a bit. In particular, `PARTICLE_HANDLERS` looks like the actual registry, hence my confusion.
Would it make sense (and not break anything) to just rename this class `RegisteredParticleFileHandler` ?

Reply:
Should be fixed now :)

Reply:
thanks !